src/python/pants/backend/project_info/list_roots_test.py

```diff
@@ -109,6 +109,7 @@ class RootsTest(GoalRuleTestBase):
     def rules(cls):
```

Comment:
Should this goal even exist anymore? It's just parroting what `--root-options` is configured to, which is the exact same thing as running `./pants options | grep 'root_options'` afaict.

Reply:
I think it should because my next change will be to allow custom source root logic (via sentinel files, or whatever).